common/user/user.go

```diff
@@ -0,0 +1,98 @@
+package user
+
+import (
+	userv1 "github.com/openshift/api/user/v1"
+	workshopv1 "github.com/stakater/workshop-operator/api/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// NewUser create an user
+func NewUser(workshop *workshopv1.Workshop, scheme *runtime.Scheme, username string, labels map[string]string) *userv1.User {
+
+	user := &userv1.User{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   username,
+			Labels: labels,
+		},
+		FullName: username,
+	}
+	return user
+}
+
+// NewRoleBindingUser create a RoleBinding for user
+func NewRoleBindingUser(workshop *workshopv1.Workshop, scheme *runtime.Scheme, username string, namespace string,
```

Comment:
@MaazMS rename it to `NewUserRoleBinding` it's important to have proper names

Reply:
I have updated `NewRoleBindingUser` method name to `NewUserRoleBinding` method name.